templates/repo/diff/box.tmpl

```diff
@@ -24,7 +24,7 @@
 			{{end}}
 		</div>
 		<div class="diff-detail-actions">
-			{{if and .PageIsPullFiles $.SignedUserID (not .IsArchived) (not .DiffNotAvailable)}}
```

Comment:
Why was this `IsArchived` and the one below removed?

Reply:
The old code `.IsArchived` is not right and does nothing.

* This one: it only shows the "reviewed" information`, it isn't really affected no matter archived or not.
* Below one: `new_review.tmpl` could handle the "archived" correctly